### ISSUE TITLE
Simple find/replace system

### ISSUE THREAD

**Issue by Bluscream** [ReSpeak/TS3Hook#17]
```ini
find=Windows,3.1.7\s[Build:\s1513163251]
replace=Linux,3.1.8\s[Build:\s1513163251]
```

**Comment by Najsr** [comment]
I've been looking into this but my client crashes after I try to change _client_version_sign_.   
Don't you know what could it cause?

**Comment by Najsr** [comment]
I've been looking into this for a while but I can't figure out why it isn't working. 
 https://github.com/Najsr/TS3Hook/commit/6a628a75e9f625ac42c262a0eb08e15c4fcbd580 Default values (from my client) work fine but whenever I change something client crashes when it tries to connect to any server.

**Comment by Splamy** [comment]
-have you checked that all values get found?
-Keep in mind that you cant make the message longer than the `int length` passed
-Messages which get splitted pass the log_out method multiple times, one time in full form and then each split packet.

**Comment by Najsr** [comment]
The problem was in length, thank you very much :)

**Comment by VisualStudio-zz** [comment]
This would be really nice, maybe something where you can find a command and you can set the parameters on it like:
`
find=clientupdate
parameters=client_badges=overwolf=0:badges=(guid's here)
`

**Comment by Bluscream** [comment]
If you search something to set your badges, you might want to check out https://r4p3.net/resources/custom-badges-aio.171/

**Comment by Undefined3301** [comment]
Hi there,

I actually tried to use the trick with my actual configuration of TeamSpeak, i use beta version and it is the 1.3.8 version of TeamSpeak.

I tried with adding this piece into the HookConf.ini file :
`find=Windows,3.1.8\s[Build:\s1516614607]
replace=Linux,3.1.9\s[Build:\s1516614607]`

(with the timestamp written in the About section of my Teamspeak Client...)

Am i doing things correctly on it just got patched on the new Teamspeak version :O ?


Thanks in advance for your assistance.

**Comment by Bluscream** [comment]
@Undefined3301 That's not even implemented (yet)

**Comment by Undefined3301** [comment]
@Bluscream Oh ok i see thanks for the reply :) (hope that we will someday be able to have a custom version of TeamSpeak ( ͡° ͜ʖ ͡°)).

Have a nice day :D.

**Comment by Bluscream** [comment]
@Undefined3301 You can use #18 to change your version.

**Comment by Bluscream** [comment]
Theres no 3.1.9, you could not use that on official servers. `3.1.8 [Build: <somenumber>]|Linux|<version hash>` would work

**Comment by Undefined3301** [comment]
@Bluscream You're too fast at answering XD, so, something like : 
`clientversion=realversion [Build: TimestampOfTheReleaseDate]|Platform|RandomStuff ? `
would work ?

**Comment by Bluscream** [comment]
I updated my comment

**Comment by Undefined3301** [comment]
@Bluscream Ok thank you mate :D

**Comment by Undefined3301** [comment]
@Bluscream Awww, strange, i used the exact launch option you gave me, i checked it twice from the version.csv file, but it looks like it does not work for me, so, what did you mean by "official servers" ?
Is there something wrong with that .ini file ?
![image](https://user-images.githubusercontent.com/26310745/35299909-242bb5a2-0087-11e8-8680-93cd13d9472c.png)

**Comment by Bluscream** [comment]
Nah the inifile looks fine to me

**Comment by Undefined3301** [comment]
@Bluscream 
![image](https://user-images.githubusercontent.com/26310745/35300076-b2c87f5c-0087-11e8-860d-9583f94f451d.png)

Same from a different tab connected to the same server with different identity :thinking:

**Comment by Bluscream** [comment]
It should show for yourself too. Hmm are you using the latest release?

**Comment by Undefined3301** [comment]
@Bluscream Yes, i'm on the beta channel, so maybe the last beta update did something to break this ? 
I didn't tested with a 3.1.7 stable version, should i ?

**Comment by Bluscream** [comment]
It works for me with 3.1.8 stable

**Comment by Undefined3301** [comment]
@Bluscream Ok, actually i switched to the 3.1.8 stable version, when i try to connect now the client is crashing 😂

**Comment by Bluscream** [comment]
Top kek, you should check your privileges

**Comment by Undefined3301** [comment]
I'll reinstall a stable version from the official website, i just switched from beta to stable channel from the settings so maybe it causes a problem, give me time to download again ...

**Comment by Undefined3301** [comment]
@Bluscream 
Ok, i finally figured it out, it was because i had an older TS3Hook version (1.1.1), i just need to update it :D.
Thanks for your time btw <3 !

**Comment by Bluscream** [comment]
I was asking that in https://github.com/ReSpeak/TS3Hook/issues/17#issuecomment-359927233 kek
